src/test/regress/sql/null_parameters.sql

```diff
@@ -0,0 +1,313 @@
+-- this file contain tests with pruning of NULL
+-- values with prepared statements
```

Comment:
we can consider making this file smaller, but it only takes around 900msecs on my local, so I'm ok to keep